pages/glossary/section-of-repeated-content.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Section of repeated
+key: section-of-repeated-content
+unambiguous: true
+objective: false
+---
+
+A [section of content][] of an [HTML web page][html web page] is a _section of repeated content_ if there exists almost identical [sections of content][] in other [HTML web pages][html web page] in a given set of [HTML web pages][html web page].
```

Comment:
I don't think "almost identical" is unambiguous :-)

Reply:
What about (taking inspiration from your lenghty explanation!)

A [section of content][] of an [HTML web page][html web page] is a _section of repeated content_ if there exists [sections of content][] in other [HTML web pages][html web page] in a given set of [HTML web pages][html web page] that fulfil the same key purpose. 

(or perhaps "that fulfil the same user expectations").